server/controllers/auth-controller.js

```diff
@@ -0,0 +1,40 @@
+const bcrypt = require("bcryptjs");
```

Comment:
Nit : I believe `require()` is an old school way of including package so if possible then we should use `imports` to include packages.

Reply:
I believe it is still used in modern projects. If you want then I can switch to newer way to imports.

Reply:
That would be great!